docs/library-hours/api.md

```diff
@@ -5,7 +5,50 @@ title: Library Hours
 Library Hours is an application that allows Stanford Libraries to set hours of operation for various libraries on campus. It also provides an API in which consumers can query information about the hours of operations.
 
 
-## API
+## JSON API
```

Comment:
```suggestion
## API
```
Should this just be API and standard across services? 🤷‍♂

Reply:
🤷‍♀️

Reply:
🤷‍♂ 🤷‍♂ 🍤 🐧